cli.php

```diff
@@ -11,6 +11,10 @@ class WPSuperCache_Command extends WP_CLI_Command {
 	 * @synopsis [--post_id=<post-id>] [--permalink=<permalink>]
 	 */
 	function flush( $args = array(), $assoc_args = array() ) {
+		require_once( WPCACHEHOME . '/wp-cache-phase1.php' );
+		global $WPSC_HTTP_HOST;
+		$WPSC_HTTP_HOST = parse_url(get_site_url())['host'];
```

Comment:
Let's see what @danielbachhuber says.
I've had issued with `automatically deducing` constant values from variables.
Maybe there are things which are not to be fully automated.